game_eggs/FoundryVTT/README.md

```diff
@@ -0,0 +1,12 @@
+# Foundry VTT
+Foundry VTT is a standalone application built for experiencing multiplayer tabletop RPGs using a feature-rich and modern self-hosted application where your players connect directly through the browser.
+
+# Installation
+Foundry requires a license. In order to use this egg, you will need to purchase a foundry license, select the linux platform from your profile on the website, and then paste the "Timed URL" into the variable when seting up the server.
```

Comment:
Can we add a link to their site here?

Reply:
https://foundryvtt.com/

Reply:
I meant to ask if you could add it to the readme.

Reply:
> I meant to ask if you could add it to the readme.

done